### DISCUSSION TITLE
noiseSuppression

### DISCUSSION THREAD

**Discussion by solucionesuno** [InnovateAsterisk/Browser-Phone#427] [Q&A]
noiseSuppression = 1 seem to dont have any effect on a call.

how thats work? is anyway to check if it is working?

thanks!!!!

**Comment by InnovateAsterisk** [comment]
This is the reference page:
https://developer.mozilla.org/en-US/docs/Web/API/MediaTrackSupportedConstraints/noiseSuppression

Maybe its not notable on your equipment? maybe its to do with the type of microphone. 

Personally I have never had "bad" (or noisy) microphone quality

**Comment by solucionesuno** [comment]
Thansk for your quick replay!

The problem is that we are using the phone browser in a call center, with a lot of room ambient noise of another operators.

i think that the problem also is related with the gain of the microphone. 

is no way to adjust manually the gain of the microphone by code or manually really? always is using the gain microhpnoe from widodws?

thanks!

**Reply by InnovateAsterisk** [reply]
I would really try a good quality headset. I'm not sure what you are currently using now, but I have experienced a huge difference from using good hardware. A lot of the time the devices have noise-cancelling built in.
